pvlib/pvsystem.py

```diff
@@ -534,11 +534,11 @@ def retrieve_sam(name=None, samfile=None):
         name = name.lower()
 
         if name == 'cecmod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2015-6-30.csv'
         elif name == 'sandiamod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2014-1-14.csv'
-        elif name == 'sandiainverter':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-inverters-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2015-6-30.csv'
+        elif name == 'sandiainverter': # Still only one current inverter set, so left it as sandia, to avoid breaking old code
```

Comment:
Ok, good points.

The doc string needs to be updated with something about how you can use either name and there is currently no difference between the two. I will merge after that.